recipe/recipe.yaml

```diff
@@ -17,8 +17,8 @@ source:
   # compare the exceedingly clear instructions/pleas here:
   # https://github.com/lightvector/KataGo/blob/master/Compiling.md#building-for-distributed
   git: https://github.com/lightvector/KataGo.git
-  tag: v${{ version }}
-  depth: 0
+  # tag: v${{ version }}
+  rev: cd0ed6c0712088ddb901be68189ba7fa1439a9e7
```

Comment:
Can it clone `HEAD` if we have nothing?

```suggestion
```

Admittedly we would want to checkout a tag later, but that is ok

Reply:
Trying to checkout during build now.

Reply:
Even `HEAD` fails on Windows.

Reply:
Moving to https://github.com/conda-forge/katago-feedstock/pull/11#issuecomment-2608733863